src/containers/PageContainer.jsx

```diff
@@ -26,6 +26,22 @@ class PageContainer extends React.PureComponent {
     this.dispatch.updatePageId(id);
   }
 
+  shouldDisable = questions => {
+    const { answers } = this.props;
+    if (questions === undefined) {
+      return false;
+    }
```

Comment:
good on you for thinking of this case, however, `questions === null` when there are no questions on the page so this catch won't quite work as you would expect. try this instead: `if (!questions) return false;`